packages/my-near-wallet/src/lib/my-near-wallet.ts

```diff
@@ -16,6 +16,7 @@ import type {
 } from "@near-wallet-selector/core";
 import { createAction } from "@near-wallet-selector/wallet-utils";
 import icon from "./icon";
+import edgeSSRLoader from "next/dist/build/webpack/loaders/next-edge-ssr-loader";
```

Comment:
Why is this import statement needed here?

Reply:
I didn't realize that. Thank you!